### PR TITLE
[DRFT-872] Add HSTS header in all responses

### PR DESCRIPTION
**Pull request by jpramos123**
## Secure Coding Practices Checklist Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [x] General Coding Practices

### CODE REVIEW

File: run_app_locally.sh [hunk 1]
```diff
@@ -32,9 +32,10 @@ if [ -z "$ACG_CONFIG" ]; then
 else
   echo "Found ACG_CONFIG - RUNNING WITH CLOWDER"
   export prometheus_multiproc_dir=$TEMPDIR
+  export LOG_LEVEL='debug'
```
Comment (vkrizan) [inline run_app_locally.sh:35]: Isn't `LOG_LEVEL` already set via env var? Does it need to be debug?

File: run_app_locally.sh [hunk 2]
```diff
@@ -32,9 +32,10 @@ if [ -z "$ACG_CONFIG" ]; then
...
   PORT=8001
   METRICS_PORT=9001
   APP_CONFIG='gunicorn.conf.py'
-  exec gunicorn wsgi --bind=0.0.0.0:$PORT --bind=0.0.0.0:$METRICS_PORT --access-logfile=- --config "$APP_CONFIG"
+  exec gunicorn wsgi --reload --bind=0.0.0.0:$PORT --bind=0.0.0.0:$METRICS_PORT --log-level=$LOG_LEVEL --access-logfile=- --config "$APP_CONFIG"
```
Comment (vkrizan) [inline run_app_locally.sh:39]: I've noticed that the variables lack quotation on `$LOG_LEVEL` and `$PORT`.

File: drift/views/v1.py [hunk 3]
```diff
@@ -334,3 +334,15 @@ def ensure_account_number():
     message = "Validating account number"
     current_app.logger.audit(message, request=request)
     return view_helpers.ensure_account_number(request, current_app.logger)
+
+
+@section.after_app_request
+def ensure_hsts_response(response):
```
Comment (vkrizan) [inline drift/views/v1.py:340]: Suggestion: This might be made global for the whole app (not only v1).

Comment (jpramos123) [reply drift/views/v1.py]: Agree! Made the changes to the code!

**Comment by jpramos123** [comment]
@johnsonm325 I made the changes suggested by Viliam, can you take a look and approve it accordingly?

Thanks

**Comment by sthirugn** [comment]
viliam is out and not back until Monday. can you check with python developers from other teams?

**Review by johnsonm325 — APPROVED** [review]

**Pull request merged**